virt/support/monitoring/virt-monitoring-vm-health.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id="virt-monitoring-vm-health"]
+= Virtual machine health checks
+include::_attributes/common-attributes.adoc[]
+:context: virt-monitoring-vm-health
+
+toc::[]
+
+You can configure virtual machine (VM) health checks by defining readiness and liveliness probes in the `VirtualMachine` resource.
+
+include::modules/virt-about-readiness-liveness-probes.adoc[leveloffset=+1]
+
+include::modules/virt-define-http-readiness-probe.adoc[leveloffset=+2]
+
+include::modules/virt-define-tcp-readiness-probe.adoc[leveloffset=+2]
+
+include::modules/virt-define-http-liveness-probe.adoc[leveloffset=+2]
+
+[id="watchdog_{context}"]
+== Defining a watchdog
+
+You can define a watchdog to monitor the health of the guest operating system (OS) by performing the following steps:
+
+. Configure a watchdog device for the virtual machine (VM).
+. Install the watchdog agent on the guest.
+
+The watchdog device monitors the agent and performs one of the following actions if the guest OS is unresponsive:
+
+* `poweroff`: The VM powers down immediately. If `spec.running` is set to `true` or `spec.runStrategy` is not set to `manual`, then the VM reboots.
+* `reset`: The VM reboots in place and the guest OS cannot react.
++
+This option is not recommended because the reboot time might cause liveliness probes to time out. If cluster-level protections detect a failed liveliness probe, the VM might be forcibly rescheduled, increasing the reboot time.
```

Comment:
I think it would look a little weird if we wrote, "Do not use this option". :-)  However, your comment is valid and I will try to find a different way to word it.